controls/pcidss_ocp4.yml

```diff
@@ -19,14 +19,16 @@ controls:
   levels:
   - base
   notes: ''
-  status: not applicable
+  status: automated
```

Comment:
Can you explain to me the relation between the status in the section header and the individual rules? In this section, it seems we have most not applicable and one automated, but in section 1.3, we have some automated rules, but the status of the section 1.3 as a whole is 'pending'.

Reply:
When I proposed the status I actually didn't think about control groups. Given that only one of the sub-controls belonging to this group was automatable and is implemented, I decided to mark this whole group as `automated`. ideas or alternatives are welcome.

Reply:
OK,  this works as long as we are consistent